packages/helium-entity-manager-sdk/src/pdas.ts

```diff
@@ -112,6 +112,21 @@ export function decodeEntityKey(
   }
 }
 
+export const keyToAssetKeyRaw = (
```

Comment:
Pointless function? Not used anywhere in this PR

Reply:
useful to have, we're going to need it later